.github/workflows/release.yml

```diff
@@ -0,0 +1,46 @@
+name: Tag & Release
+
+on:
+  push:
+    branches:
+      - 'main'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.9'
+
+      - name: Install dev dependencies
+        run: python -m pip install --upgrade pip setuptools poetry
+
+      - name: Build binary wheel and a source tarball
+        run: poetry build
+
+      - name: Get version
+        run: echo "VERSION=$(poetry version --short)" >>$GITHUB_OUTPUT
+        id: version
```

Comment:
presumably this is the version written in the pyproject.toml file?

Reply:
yep